models/value_set.go

```diff
@@ -105,7 +123,13 @@ func (v ValueSetMap) CodeDisplayWithPreferredCodeAndLaterality(entry *Entry, cod
 	if err != nil {
 		log.Fatal(err)
 	}
-	codeDisplay.PreferredCode = coded.PreferredCode(codeDisplay.PreferredCodeSets, codeDisplay.CodeSetRequired, codeDisplay.ValueSetPreferred, v, MapDataCriteria.ValueSetOid)
+	preferredCodes := coded.PreferredCodes(codeDisplay.PreferredCodeSets, codeDisplay.CodeSetRequired, codeDisplay.ValueSetPreferred, v, MapDataCriteria.ValueSetOid)
+	if len(preferredCodes) > 0 {
```

Comment:
Lines 127 to 132 pop up a lot in this file. If it was twice, I'd ignore it, but with 4 times I think it would be beneficial to put it in a function. Something like a `func (CodeDisplay) setCodeDisplayFromPreferredCodes(preferredCodes)`.